requirements.txt

```diff
@@ -2,11 +2,12 @@ blinker==1.7.0
 click==8.1.7
 colorama==0.4.6
 Flask==3.0.2
-iniconfig==2.0.0
 itsdangerous==2.1.2
 Jinja2==3.1.3
 MarkupSafe==2.1.5
-packaging==24.0
-pluggy==1.4.0
-pytest==8.1.1
-Werkzeug==3.0.1
```

Comment:
Why were some of these requirements removed?

Im not sure if some of these are dependencies of others. At the least Pytest should be kept.

Reply:
I have no idea.  I will make sure that they are included for final merge